hazelcast-spring/src/main/resources/hazelcast-spring-5.2.xsd

```diff
@@ -3830,6 +3831,13 @@
         </xs:restriction>
     </xs:simpleType>
 
+    <xs:complexType name="btree-index">
+        <xs:all>
+            <xs:element name="page-size" type="memory-size" minOccurs="1"/>
```

Comment:
Why does `minOccurs="1"`? Why don't we let the user omit this value if they want to use the default value for it? Also, can you add the documentation in hazelcast-config-5.2.xsd to here as well?